### PR TITLE
Support for Symfony 4

### PR DESCRIPTION
**Pull request by sroze**
Use this opportunity to also drop Symfony 2.3, replace 3.0 with 3.3 in Travis.

Depends on:
- [x] https://github.com/Behat/MinkExtension/pull/308
- [x] https://github.com/minkphp/MinkBrowserKitDriver/pull/123
- [x] https://github.com/minkphp/Mink/pull/749

### CODE REVIEW

**Comment by sroze** [comment]
The remaining is mink's dependencies 🎉 
- [ ] https://github.com/Behat/MinkExtension/pull/308
- [ ] https://github.com/minkphp/MinkBrowserKitDriver/pull/123

**Comment by tyx** [comment]
I'm available to help on this subject as this PR is the latest requirement to bump to symfony 4 on our side ^^

**Comment by sroze** [comment]
I believe the last PR is the one on minkphp/MinkBrowserKitDriver#123

File: .travis.yml [hunk 1]
```diff
@@ -14,22 +14,19 @@ cache:
...
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION='3.4.*'
+    - php: 7.1
+      env: DEVS=dev SYMFONY_VERSION='4.0.x-dev'
   allow_failures:
-    - php: 7.0
-      env: SYMFONY_VERSION='3.0.*'
     - php: 7.1
-      env: SYMFONY_VERSION='3.0.*'
+      env: SYMFONY_VERSION='4.0.x-dev'
```
Comment (ruudk) [inline .travis.yml]: Maybe change this to `4.0.*`?

File: .travis.yml [hunk 2]
```diff
@@ -14,22 +14,19 @@ cache:
...
       env: SYMFONY_VERSION='2.8.*'
-    - php: 7.0
-      env: DEPS=dev SYMFONY_VERSION='3.0.*'
     - php: 7.1
-      env: DEPS=dev SYMFONY_VERSION='3.0.*'
+      env: SYMFONY_VERSION='3.3.*'
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION='3.4.*'
+    - php: 7.1
+      env: DEVS=dev SYMFONY_VERSION='4.0.x-dev'
```
Comment (ruudk) [inline .travis.yml]: Maybe change this to 4.0.*?

File: .travis.yml (hunk 1 above)
Comment (sroze) [reply .travis.yml]: Yep 👍

File: .travis.yml (hunk 2 above)
Comment (sroze) [reply .travis.yml]: Yep.

File: .travis.yml (hunk 1 above)
Comment (sroze) [reply .travis.yml]: Not here, these are the allowed failures

**Comment by sroze** [comment]
Merged and relased [2.1.3](https://github.com/Behat/Symfony2Extension/releases/tag/2.1.3) 🎉

**Comment by ruudk** [comment]
@sroze You rock! 💪

**Pull request merged**